contracts/sei-tester/src/contract.rs

```diff
@@ -49,6 +49,7 @@ pub fn place_orders(
         position_direction: PositionDirection::Long,
         order_type: OrderType::Limit,
         data: "".to_string(),
+        status_description: "".to_string(),
```

Comment:
what is status_description used for? if it will have a limited number of values maybe lets us an enum?

Reply:
this is what we discussed during the bug bash. We need to allow contract to set arbitrary strings to inform users why their orders have failed to place. Enums don't make much sense as it would require sei to know contracts' product specifics.